src/BenchmarkDotNet/Running/BuildPartition.cs

```diff
@@ -46,6 +46,8 @@ public BuildPartition(BenchmarkBuildInfo[] benchmarks, IResolver resolver)
             // given job can have CoreRT toolchain set, but Runtime == default ;)
             || (RepresentativeBenchmarkCase.Job.Infrastructure.TryGetToolchain(out var toolchain) && toolchain is CoreRtToolchain);
 
+        public bool IsWasm => Runtime is WasmRuntime;
```

Comment:
we should handle the edge case where some of our users did set Toolchain to Wasm, but forgot to set Runtime to Wasm as well

```suggestion
        public bool IsWasm => Runtime is WasmRuntime;
            // given job can have Wasm toolchain set, but Runtime == default ;)
            || (RepresentativeBenchmarkCase.Job.Infrastructure.TryGetToolchain(out var toolchain) && toolchain is WasmToolchain);
```